README.md

```diff
@@ -165,9 +165,15 @@ threads are handled by Android runtime.
 
 For R8 no actions required, it will take obfuscation rules from the jar.
 
-For Proguard  you need to add options from [coroutines.pro](kotlinx-coroutines-core/jvm/resources/META-INF/proguard/coroutines.pro) to your rules manually.
+For Proguard  you need to add options from 
```

Comment:
Actually, it's the Android Gradle Plugin that takes care of extracting rules from META-INF, so if you're compiling for Android using our Gradle build, the rules should work automatically with either ProGuard or R8
(since you're changing this file anyway...)

Reply:
I read that those rules worked only for R8. Does it depend on a minimum AGP version instead?

Reply:
I reworded this part of readme.